content/en/01-about-pixie/01-what-is-pixie.md

```diff
@@ -7,32 +7,34 @@ redirect_from:
     - /about-pixie/how-pixie-works
 ---
 
-Pixie is an open source observability tool for Kubernetes applications. Pixie uses eBPF to automatically capture its telemetry data without the need for manual instrumentation.
+Pixie is an open source observability tool for Kubernetes applications.
 
-Using Pixie, developers can view the high-level state of their cluster (service maps, cluster resources, application traffic) and also drill-down into more detailed views (pod state, flame graphs, individual application requests).
+Linux [eBPF](http://www.brendangregg.com/ebpf.html) is used to automatically capture telemetry data without the need for manual instrumentation.
```

Comment:
i link the linking but not sure we need to make this a separate line, the first sentence is a bit vague on its own